packages/dd-trace/src/ci-visibility/intelligent-test-runner/get-skippable-suites.js

```diff
@@ -19,7 +19,7 @@ function getSkippableSuites ({
     headers: {
       'Content-Type': 'application/json'
     },
-    timeout: 15000,
+    timeout: 20000,
```

Comment:
This sounds like a long time, and it also sounds like it's based on the size of the repository. Couldn't a very large repository take even more time? And then some more? Why does this take so long to begin with and can we make it faster?